crates/re_data_store/benches/gc.rs

```diff
@@ -156,8 +157,9 @@ where
     for _ in 0..NUM_ROWS_PER_ENTITY_PATH {
         #[allow(clippy::needless_range_loop)] // readability
         for i in 0..NUM_ENTITY_PATHS {
-            let row = rows_per_table[i].next().unwrap();
-            store.insert_row(&row.unwrap()).unwrap();
+            if let Some(Ok(row)) = rows_per_table[i].next() {
+                store.insert_row(&row).ok_or_log_error();
+            }
```

Comment:
this file is part of a benchmark suite, so it _should_ count as testing code and allow unwraps 🤔 
This is likely not very important but I'm worried by swallowing errors here when they mean that the benchmark/test broke down and we'd rather have this crash

Reply:
Makes sense! Sorry, I'm new to the code base so I missed that.

Reply:
np!